go-common.mk

```diff
@@ -164,6 +164,12 @@ endif # GORUNGET
 
 $(GOTESTCOVERHTML): $(GOTESTCOVERRAW)
 	$(GO) tool cover -html=$< -o $@
+	# To allow Code Climate to understand our uploaded coverage files
+	case "$$(grep '^module' go.mod | awk -F/ '{print$$NF}')" in \
+		v[0-9]) \
+			sed -i'' -e 's!/v[0-9]/!/!' "$(GOTESTCOVERRAW)" "$(GOTESTCOVERHTML)" \
```

Comment:
This only works for module versions up to v9.  If we need to do v10 of a module then it would need to change.  That might be a little deliberate friction about making backwards incompatible changes something we should think through fairly carefully and not do that often.